app/routes.py

```diff
@@ -1,17 +1,30 @@
-from flask import render_template, flash, redirect
+from flask import render_template, flash, redirect, jsonify
 from app import app
 from app.controllers.login import Login
+from app.controllers.post import PostController
 
 
 @app.route("/login", methods=["GET"])
 def login():
-	return Login.get()
+    return Login.get()
+
 
 @app.route("/login", methods=["POST"])
 def redirect():
-	return Login.post()
+    return Login.post()
+
+
+@app.route("/postEdit", methods=["GET"])
+@app.route("/postEdit/<int:postId>", methods=["GET"])
+def show_post_editor(postId=None):
+    return PostController.showPostEditor(postId)
+
 
+@app.route("/postEdit", methods=["POST"])
+@app.route("/postEdit/<int:postId>", methods=["POST"])
+def create_update_post(postId=None):
+    return PostController.post(postId)
 
-@app.route("/postview", methods=["GET"])
-def postview():
-    return None
+@app.route("/Post", methods=["GET"])
```

Comment:
It seems Weimeng added this; not sure why its added in this branch, and it needs to be reviewed, but I dont want to stop all the other changes for this. We will catch it as we continue working on Sunday